completions/_asdf

```diff
@@ -170,7 +170,9 @@ case "$subcmd" in
   ;;
 (latest)
   if (( CURRENT == 3 + IntermediateCount )); then
-    _asdf__installed_plugins
+    _alternative  \
```

Comment:
For `plugin-update` above I see:

```
(( CURRENT == 3 + IntermediateCount )) && _alternative \
    'all:all:(--all)' \
    ...
```

Is there a significance between the two different syntaxes?

Reply:
Not sure if I completely understand your question, but I did the `if (( )); then ...` thing instead of `(( )) && ...` because there was already an if statement like `install` and it wasn't like `plugin-update`. In the context of completion, I _think_ there is no difference, since `set -e` is not set - I wish I remembered why I avoid it in regular scripts